doc-content/enterprise-only/installation/clustering-quartz-proc.adoc

```diff
@@ -0,0 +1,115 @@
+[id='clustering-quartz-proc']
+= Configuring Quartz timer service
+
+When you run {KIE_SERVER} in a cluster you must configure the Quartz timer service.
+
+Before you configure a database on your application server, you must prepare the database for Quartz to create Quartz tables, which will hold the timer data, and the Quartz definition file.
+
+.Prerequisites
+
+* A supported non-JTA data source is connected to your application server, for example a PostgreSQL data source.
+
+.Procedure
+
+. Create Quartz tables in your database to enable timer events to synchronize using the DDL script for your database.
++
+The DDL script is available in the extracted supplementary ZIP archive in `_QUARTZ_HOME_/docs/dbTables`.
+
+. Create the Quartz configuration file `quartz-definition.properties` in the `_JBOSS_HOME_/_MODE_/configuration/` directory and add the following example content:
++
+ifdef::PAM[]
+[source, bash]
+----
+#=========================================================================
+# Configure Main Scheduler Properties
+#=========================================================================
+org.quartz.scheduler.instanceName = jBPMClusteredScheduler
+org.quartz.scheduler.instanceId = AUTO
+#=========================================================================
+# Configure ThreadPool
+#=========================================================================
+org.quartz.threadPool.class = org.quartz.simpl.SimpleThreadPool
+org.quartz.threadPool.threadCount = 5
+org.quartz.threadPool.threadPriority = 5
+#=========================================================================
+# Configure JobStore
+#=========================================================================
+org.quartz.jobStore.misfireThreshold = 60000
+org.quartz.jobStore.class=org.quartz.impl.jdbcjobstore.JobStoreCMT
+org.quartz.jobStore.driverDelegateClass=org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
+org.quartz.jobStore.useProperties=false
+org.quartz.jobStore.dataSource=managedDS
+org.quartz.jobStore.nonManagedTXDataSource=notManagedDS
+org.quartz.jobStore.tablePrefix=QRTZ_
+org.quartz.jobStore.isClustered=true
+org.quartz.jobStore.clusterCheckinInterval = 20000
+#=========================================================================
+# Configure Datasources
+#=========================================================================
+org.quartz.dataSource.managedDS.jndiURL=jboss/datasources/psbpmsDS
+org.quartz.dataSource.notManagedDS.jndiURL=jboss/datasources/quartzNotManagedDS
+# Note the configured data sources that accommodate the two Quartz schemes at the very end of the file.
+----
+endif::PAM[]
+ifdef::DM[]
```

Comment:
quartz is only used by PAM and should not be then mentioned in DM docs

Reply:
Done, removed the DM content and added PAM constraint